### PR TITLE
Gradle tests cleanup

### PR DESCRIPTION
**Pull request by dstepanov**
Found a new way :relieved:

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -134,51 +143,60 @@ project(':accurest-gradle-plugin') {
 		testCompile project(':accurest-testing-utils')
 	}
 
+	configurations {
```
Comment (marcingrzejszczak) [inline build.gradle:146]: why 2 separate `configurations` blocks?

**Comment by marcingrzejszczak** [comment]
So basically you're copying the jars to 2 folders and then you're passing the link to those folders via a system property? Nice!

**Comment by dstepanov** [comment]
BTW Why is there funcTest which runs just Basic spec? Can we merge it to test?

**Comment by marcingrzejszczak** [comment]
@szpak do you remember why we have this `funcTest` ?

**Comment by dstepanov** [comment]
> So basically you're copying the jars to 2 folders and then you're passing the link to those folders via a system property? Nice!

Yes!

**Comment by dstepanov** [comment]
Cosmetic changes

**Comment by marcingrzejszczak** [comment]
Great job @dstepanov !

**Pull request merged**